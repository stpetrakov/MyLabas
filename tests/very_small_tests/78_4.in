32356 19952 4787 28970 15025 11717 10813 9643 19219 5428 17263 8147 28313 19169 16110 24584 32042 31719 14072 29404 32536 17691 29594 25721 25336 24980 28451 834 17842 8811 32472 13751 212 5055 29673 26521 29484 29436 18596 21071 14229 23133 11753 9353 15889 17840 3294 19747 31512 21594 20707 27980 8496 27802 3529 13799 6334 9234 22078 13361 10780 4483 22156 20901 3563 27608 1924 2845 22443 6758 3508 9017 28297 6770 6369 21508 2167 24051 