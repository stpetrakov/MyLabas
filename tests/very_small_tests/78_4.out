212 834 1924 2167 2845 3294 3508 3529 3563 4483 4787 5055 5428 6334 6369 6758 6770 8147 8496 8811 9017 9234 9353 9643 10780 10813 11717 11753 13361 13751 13799 14072 14229 15025 15889 16110 17263 17691 17840 17842 18596 19169 19219 19747 19952 20707 20901 21071 21508 21594 22078 22156 22443 23133 24051 24584 24980 25336 25721 26521 27608 27802 27980 28297 28313 28451 28970 29404 29436 29484 29594 29673 31512 31719 32042 32356 32472 32536 